Language/fa/Time.php

```diff
@@ -29,5 +29,5 @@
     'inFuture'       => 'در {0}',
     'yesterday'      => 'دیروز',
     'tomorrow'       => 'فردا',
-    'now'            => 'هیمن حالا',
+    'now'            => 'همین حالا',
```

Comment:
```suggestion
    'now'            => 'اکنون',
```
What about this change?

Reply:
> How about this change?

its better